extension/src/stabilization_info.rs

```diff
@@ -11,6 +11,17 @@
 
 crate::functions_stabilized_at! {
     STABLE_FUNCTIONS
+    "1.12.0" => {
+        stats1d_tf_combine(internal,internal),
+        stats1d_tf_inv_trans(internal,double precision),
+        stats1d_tf_final(internal),
+        stats1d_tf_trans_deserialize(bytea,internal),
+        stats1d_tf_trans(internal,double precision),
+        stats2d_tf_final(internal),
+        stats1d_tf_trans_serialize(internal),
+        stats2d_tf_trans(internal,double precision,double precision),
```

Comment:
Looks like you're missing the combine, serialize, and deserialize for the stats2d.  I also see we have `stats1d/2d_summary_trans` and `stats1d/2d_summary_inv_trans` which we use for rollup, would it make sense to have these use TwoFloat as well?

Reply:
The combine/serialize/deserialize functions aren't used in moving aggregate mode (since moving aggregate mode doesn't support being parallelized) – I think it might be better to remove them for stats1d instead of adding them to stats2d?

TwoFloat summaries are never exposed to PostgreSQL (the final function converts TwoFloat summaries to normal single-float summaries) so they can't be rolled up.

Reply:
We can't remove them for stats1d because in normal mode they can still be parallelized? Or maybe I'm missing something?

Reply:
(by normal mode I mean when the aggregate is not used  in window function)

Reply:
An aggregate can be in partial aggregation mode, or moving aggregation mode, but not both at the same time. The serial/deserial/combine functions are never called in moving aggregate mode (and there is no way to specify such functions that would never be called). The only reason they exist currently is for the perma-experimental `stats_agg_tf` (which always uses twofloats for the aggregate state).

The serial/deserial/combine functions are already only used in the `stats_agg_tf` function for testing – removing them would only affect the performance of that experimental function, which I wouldn't expect anyone to be relying on.

Reply:
Aha! Okay. got it, thanks for the explanation! We might want to add a comment about that in the code somewhere so that when we inevitably forget this in 3 weeks, we don't have to reconstruct our logic quite as much :)